back/config/initializers/rack_attack.rb

```
@@ -0,0 +1,40 @@
+  end
+
+  # Custom response for throttled requests
+  self.throttled_responder = lambda do |env|
+    retry_after = (env["rack.attack.match_data"] || {})[:period]
+    [
+      429,
+      {
+        "Content-Type" => "application/json",
+        "Retry-After" => retry_after.to_s
+      },
+      [{ error: "Rate limit exceeded. Please try again later." }.to_json]
```

RuboCop (1.82.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)